modules/op-release-notes-1-15.adoc

```diff
@@ -344,6 +344,8 @@ Use the alternate parameter specs only if you need to use the additional setting
 
 * With this update, you can no longer use the `jaeger` exporter for OpenTelemetry tracing. You can use the `oltptraceexporter` for tracing.
 
+* With this update the `tls_hostname_override` option is deprecated and can no longer be used with the {tekton-results} deployment.
```

Comment:
@khrm this was deprecated already in 1.15? not 1.16?

Reply:
It was part of operator v0.71.0. It's already in 1.15.